docs/toc.html

```diff
@@ -142,16 +142,17 @@
         </li>
         <li><a href="/{{version}}/documentation/streams">9. Kafka Streams</a>
             <ul>
-                <li><a href="/{{version}}/documentation/streams#streams_overview">9.1 Overview</a></li>
-                <li><a href="/{{version}}/documentation/streams#streams_concepts">9.2 Core Concepts</a></li>
-                <li><a href="/{{version}}/documentation/streams#streams_architecture">9.3 Architecture</a></li>
-                <li><a href="/{{version}}/documentation/streams#streams_developer">9.4 Developer Guide</a></li>
+                <li><a href="/{{version}}/documentation/streams/quickstart">9.1 Quickstart</a></li>
```

Comment:
Are these TOC changes still matching the latest changes to the AK docs (cf. what Derrick did)?